sycl/test/usm/multictxtcpu.cpp

```diff
@@ -0,0 +1,28 @@
+// REQUIRES: cpu,gpu
+// RUN: %clangxx -fsycl %s -o %t1.out -lOpenCL
+// RUN: %t1.out
+
+//==----------------- multictxtcpu.cpp - Multi Context USM CPU test --------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+
+using namespace cl::sycl;
+
+int main() {
+  queue gpu_q(gpu_selector{});
```

Comment:
It seems gpu queue is not used here.

Reply:
This is testing that USM things don't break in the presence of multiple contexts.  The context just needs to exist here.

Reply:
Not sure I understand how presence of context from 1 platform can affect on work with context from another platform from design point of view.
Could you please put comment here describing it, otherwise somebody will remove it eventually as dead code.

Reply:
USM used to decide whether it was using native support or emulated support based on the first context it saw.  These tests were the simple examples that showed how this was a problem as if the CPU queue was created first, the emulated path was used even for the GPU (which didn't support it.) These tests are basically sanity checks that nothing has broken.  I can add comments to that effect.